src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -313,11 +314,18 @@
     Favicon loadFavicon(BufferedImage image) throws IOException;
 
     /**
-     * Retrieves the RecipeRegistry for this GameRegistry.
+     * Retrieves the CraftingRegistry for this GameRegistry.
+     *
+     * @return The crafting recipe registry
+     */
+    CraftingRegistry getCraftingRegistry();
+
+    /**
+     * Retrieves the SmeltingRegistry for this GameRegistry.
      *
-     * @return The recipe registry
+     * @return The smelting registry
      */
-    RecipeRegistry getRecipeRegistry();
```

Comment:
This needs to be left in with `@Deprecated` to just redirect to the `CraftingRegistry`.